### PR TITLE
README Perfection

### PR DESCRIPTION
**Pull request by Ghostamoose**
Reworded sections containing more technical wording to be more readable and contain less irrelevant information.

**Should** be the last change made to the README, barring any further changes requested by @Meorawr or @Solanya.

### CODE REVIEW

**Review by Meorawr — CHANGES_REQUESTED** [review]
I feel like most of the additions are just adding pointless fluff. Will need to re-read and think about it later.

**Review by Meorawr — APPROVED** [review]
> by combining powerful customization features with a user interface focused on usability and accessibility, along with many other features requested by our users

I still hate this bit but I can't remember what we decided to reword it to.

**Review by Meorawr — APPROVED** [review]
quest complete

**Comment by Seleves** [comment]
> quest complete

almost complete *duck*

> A customizable toolbar with useful actions like **switching from switching from** one profile to another
